src/harness/fourslashInterfaceImpl.ts

```diff
@@ -745,108 +745,154 @@ namespace FourSlashInterface {
     }
 
     interface Classification {
-        classificationType: ts.ClassificationTypeNames;
-        text: string;
+        classificationType: ts.ClassificationTypeNames | number;
+        text?: string;
         textSpan?: FourSlash.TextSpan;
     }
-    export namespace Classification {
-        export function comment(text: string, position?: number): Classification {
+
+    export function classification(format: "original" | "2020") {

```

Comment:
I don’t see much advantage in making this a factory function instead of just two namespaces since they share literally no code, but 🤷‍♂️